src/Microsoft.AspNetCore.AzureKeyVault.HostingStartup/HostingStartupConfigurationExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.AspNetCore.Hosting
+{
+    internal static class HostingStartupConfigurationExtensions
+    {
+        public static IConfiguration GetBaseConfiguration()
+        {
+            return new ConfigurationBuilder()
+                .AddEnvironmentVariables(prefix: "ASPNETCORE_")
+                .Build();
+        }
+        public static bool IsEnabled(this IConfiguration configuration, string hostingStartupName, string featureName)
+        {
+            if (configuration.TryGetOption(hostingStartupName, featureName, out var value))
```

Comment:
Replace the whole method with this trivial expression:

```c#
return !(configuration.TryGetOption(hostingStartupName, featureName, out var value) && (!(new[] { "false", "0" }.Contains(value, StringComparison.OrdinalIgnoreCase))));
```

(Just kidding, no idea if that's correct :trollface:)